eng/publish-coverlet-result-files.yml

```diff
@@ -25,6 +24,14 @@ steps:
     Contents: '**/*.trx'
     TargetFolder: '$(Build.SourcesDirectory)/artifacts/TestLogs'
 
+- task: CopyFiles@2
+  displayName: Copy binlog files
+  condition: always()
+  inputs:
+    SourceFolder: '$(Build.SourcesDirectory)'
+    Contents: '**/*.binlog'
+    TargetFolder: '$(Build.SourcesDirectory)/artifacts/TestLogs'
```

Comment:
What about copying the build logs to another directory? Maybe sth. like `../artifacts/BuildLogs`?